### DISCUSSION TITLE
Redesign the Editor to be centered horizontally against the whole screen

### DISCUSSION THREAD

**Discussion by Skubers** [ppy/osu#24207] [Ideas] [answered]
Right now, because all different editor sections have various width (See blue lines for Timeline section width, and yellow lines for side bars), nothing can be centered correctly which is pretty bad.

![image](https://github.com/ppy/osu/assets/82095491/74c35a31-5724-4a3b-9081-29b3a41b1faf)

Notice how the actual center of the editor lands on the red line, which is not even the center of the playfield (red rectangle), nor the center of the timeline (the red circle).

I would want to suggest to fix the widths of blue lines and yellow lines sections to be the same width on both sides of the screen, so Editor can actually be centered.

I can't tell you how important it is, but after using the editor for some time (transitioning from Stable editor) it definitely feels wrong.

**Comment by peppy** [comment]
I'm all for centering the playfield (it was intended to already be centered, must have regressed), but the timeline is not centered in stable, and I'm not sure how much value there is to centering it. May be more visually pleasing but usability wise I don't see it being imperative.

**Reply by peppy** [reply]
i've gotten this working anyways:

![osu Game Tests 2023-07-14 at 05 29 35](https://github.com/ppy/osu/assets/191335/3e4dcadd-aefc-451c-ae52-a0ba057043d5)

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/24220
